src/webgl/3d_primitives.js

```diff
@@ -180,6 +180,14 @@ p5.prototype.freeGeometry = function(geometry) {
   this._renderer._freeBuffers(geometry.gid);
 };
 
+/**
+ * Clears the internal Color of the Geometry.
+ * @method clearColors
+ */
+p5.prototype.clearColors = function () {
```

Comment:
I think maybe we can take out this method now that we have one on `p5.Geometry`.

Reply:
Okay